packages/material-ui/src/useAutocomplete/useAutocomplete.test.js

```diff
@@ -215,4 +215,59 @@ describe('useAutocomplete', () => {
       });
     });
   });
+
+  it('should warn if the input is not binded', function test() {
+    // TODO is this fixed?
+    if (!/jsdom/.test(window.navigator.userAgent)) {
+      // can't catch render errors in the browser for unknown reason
+      // tried try-catch + error boundary + window onError preventDefault
+      this.skip();
+    }
+
+    const Test = (props) => {
+      const { options } = props;
+      const {
+        groupedOptions,
+        getRootProps,
+        getInputLabelProps,
+        // getInputProps,
+        getListboxProps,
+        getOptionProps,
+      } = useAutocomplete({
+        options,
+        open: true,
+      });
+
+      return (
+        <div>
+          <div {...getRootProps()}>
+            <label {...getInputLabelProps()}>useAutocomplete</label>
+          </div>
+          {groupedOptions.length > 0 ? (
+            <ul {...getListboxProps()}>
+              {groupedOptions.map((option, index) => {
+                return <li {...getOptionProps({ option, index })}>{option}</li>;
+              })}
+            </ul>
+          ) : null}
+        </div>
+      );
+    };
+
+    expect(() => {
+      render(
+        <ErrorBoundary>
+          <Test options={['foo', 'bar']} />
+        </ErrorBoundary>,
+      );
+    }).toErrorDev([
+      "Error: Uncaught [TypeError: Cannot read property 'removeAttribute' of null]",
+      'Material-UI: The input ref is not binded correctly',
+      "Error: Uncaught [TypeError: Cannot read property 'removeAttribute' of null]",
+      'The above error occurred in the <ul> component',
+      'The above error occurred in the <ul> component',
+      'The above error occurred in the <Test> component',
+      'The above error occurred in the <Test> component',
```

Comment:
Some of these look like they're caused by StrictMode. This is not blocking for this PR. Just leaving a note that we need to verify what is logged in React 17.

Reply:
Ok, handled